### ISSUE TITLE
Assign requires shapes of both tensors to match

### ISSUE THREAD

**Issue by bysowhat** [tensorflow/models#3689]
InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [32] rhs shape= [256]
	 [[Node: save_1/Assign_344 = Assign[T=DT_FLOAT, _class=["loc:@ssd_512_resnet101/block3/unit_18/bottleneck_v2/conv2/BatchNorm/beta"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](ssd_512_resnet101/block3/unit_18/bottleneck_v2/conv2/BatchNorm/beta, save_1/RestoreV2_344)]]

**Comment by dheera** [comment]
are you trying to train with a different number of classes than the pretrained model? if so pass
`--initialize_last_layer=false`
to train.py

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by ymodak** [comment]
Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
